### PR TITLE
Priority annotations for interceptors.

### PR DESCRIPTION
**Pull request by JanDoerrenhaus**

### CODE REVIEW

File: cache-annotations-ri/cache-annotations-ri-cdi/src/main/java/org/jsr107/ri/annotations/cdi/CachePutInterceptor.java [hunk 1]
```diff
@@ -35,6 +36,7 @@
  */
 @CachePut
 @Interceptor
+@Priority(Interceptor.Priority.APPLICATION)
```
Comment (bannmann) [inline cache-annotations-ri/cache-annotations-ri-cdi/src/main/java/org/jsr107/ri/annotations/cdi/CachePutInterceptor.java]: switch all priorities to `LIBRARY_AFTER`, which feels more suitable

**Review by bannmann — CHANGES_REQUESTED** [review]

**Review by bannmann — APPROVED** [review]

**Review by bannmann — CHANGES_REQUESTED** [review]
we also need to bump the artifact version

**Comment by JanDoerrenhaus** [comment]
Do we, now. It was `1.1.0-SNAPSHOT` before, it is `1.1.0-SNAPSHOT` now. What do I do?

**Comment by bannmann** [comment]
Never mind. In our corporate nexus, the versions are different; they were set manually during deployment.

**Pull request merged**